modules/lang-painless/src/test/resources/rest-api-spec/test/painless/50_script_doc_values.yaml

```diff
@@ -6,9 +6,9 @@ setup:
                 mappings:
                     test:
                         properties:
-                            binary:
-                                type: binary
-                                doc_values: true
+                            #binary:
```

Comment:
should we remove all of these commented lines since it seems they have been moved to another test?